charts/finrem-ccd-definitions/values.preview.template.yaml

```diff
@@ -25,15 +25,15 @@ ccd:
       value: ${API_GATEWAY_IDAM_SECRET}
 
   userProfileApi:
-    authorisedServices: ccd_admin,ccd_data,ccd_definition,divorce
+    authorisedServices: ccd_data,ccd_definition,ccd_admin,finrem_ccd_data_migrator
```

Comment:
do we really need this fourth role? Could you please take a bit of time to make sure we only include what's really needed?
if it's needed, great. otherwise, I'd rather have minimal change.

Reply:
If we could keep the order as well, that'd make it easier to people comparing div and finrem.